presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -58,7 +59,14 @@ public Connector create(String connectorId, Map<String, String> requiredConfig)
         requireNonNull(optionalConfig, "optionalConfig is null");
 
         try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
-            Bootstrap app = new Bootstrap(new JdbcModule(connectorId), module);
+            Bootstrap app = new Bootstrap(
+                    new JdbcModule(connectorId),
+                    module,
+                    installModuleIf(
+                            JdbcSecurityConfig.class,
+                            security -> "none".equalsIgnoreCase(security.getPostgreSqlSecuritySystem()),
```

Comment:
presumably that's handled already -- the configs are in the connector-specific config file, and there are already some base JDBC configuration variables